src/workerd/jsg/modules.h

```diff
@@ -181,6 +181,10 @@ class ModuleRegistry {
       // The EXTERNAL flag tells the compile operation to treat the content as an external
       // string wrapping an immutable buffer outside the V8 heap. When this flag is not set,
       // the content is copied into the V8 heap.
+      COMPILE_CACHE,
+      // The COMPILE_CACHE flags tells the compile operation to cache compilation data for
```

Comment:
Probably worth explaining that this is a process-lifetime in-memory cache and is therefore only appropriate for built-in modules.

(I still think we should just specify the built-in-ness rather than allowing each of these options to be specified separately...)

Reply:
> (I still think we should just specify the built-in-ness rather than allowing each of these options to be specified separately...)

I agree, as I mentioned previously I was keeping these separate only because I wanted to test things individually for a bit. Now that I think definitely think it's clear we want both, combining the external and compile_cache flags into one works just fine. Will update.